chord_drs/app.py

```diff
@@ -46,9 +46,13 @@
 
 # # debugger section
 # Ensure 'debugpy' is installed (via requirements.txt or manually)
-if os.environ.get('FLASK_DEBUG', False):
-    import debugpy
-    DEBUGGER_PORT = int(os.environ.get('DEBUGGER_PORT', 5678))
-    debugpy.listen(("0.0.0.0", DEBUGGER_PORT))
-    print('Attached')
-# # end debugger section
\ No newline at end of file
+DEBUG = os.environ.get('FLASK_DEBUG', False)
+if DEBUG:
+    try:
+        import debugpy
+        DEBUGGER_PORT = int(os.environ.get('DEBUGGER_PORT', 5678))
+        debugpy.listen(("0.0.0.0", DEBUGGER_PORT))
+        print('Debugger Attached')
```

Comment:
It has readability issues in the log, among other log statements. If it would be used to check in the logs (which I assume is probably not that case, and therefore it if fine), it would be beneficial to add a newline before and after the print